package.json

```diff
@@ -28,12 +28,13 @@
   },
   "lint-staged": {
     "*.{js,css,md}": "prettier --check",
-    "*.js": "eslint"
+    "*.js": "eslint --quiet"
   },
   "dependencies": {
     "@sinonjs/commons": "^1.4.0",
     "@sinonjs/samsam": "^5.0.2",
     "array-from": "2.1.1",
+    "eslint-plugin-compat": "^3.9.0",
```

Comment:
This should be in `devDependencies`.

Reply:
It shouldn't even be here, but only in `@sinonjs/eslint-config`. I'll remove it

Reply:
This has been fixed by removing the work-in-progress commit